internal/flterrors/flterrors.go

```diff
@@ -27,6 +27,7 @@ var (
 	ErrTemplateVersionIsNil   = errors.New("spec.templateVersion not set")
 	ErrInvalidTemplateVersion = errors.New("device's templateVersion is not valid")
 	ErrNoRenderedVersion      = errors.New("no rendered version for device")
+	ErrDecommission           = errors.New("decommissioned device cannot be created or updated; to request decommissioning, use /api/v1/devices/{name}/decommission")
```

Comment:
I'd remove the second part: We don't have instructions how to solve an error anywhere else, either. Actually also: When the error is printed, the device is already decommissioned, so the instructions aren't even relevant anymore.